website/static/js/profile.js

```diff
@@ -359,6 +377,17 @@ var NameViewModel = function(urls, modes, preventUnsaved, fetchCallback) {
     self.isValid = ko.computed(function() {
         return validated.isValid();
     });
+    
+   self.nameFieldEmpty = ko.computed(function() {
+        if (self.full() === '') {
```

Comment:
This can be simplified as:

``javasript
    self.nameFieldEmpty = ko.computed(function () {
        return (self.full() === '');
    }
```

The parenthesis in the return statement are for clarity, and can be omitted at your discretion.